### DISCUSSION TITLE
Error with 'Print time' for prints over 24h?

### DISCUSSION THREAD

**Discussion by bryanseger** [sn4k3/UVtools#854] [General]
I don't often have prints that go over 24h, so I'm not sure if this is a common error among all systems or just the Jupiter that I'm currently using, but I do know that the attached print is over 24h (it's around 28 hours, I believe), but UVtools only reports it as 4 hours 3 minutes. Perhaps it's 24h + 4h3m or something along those lines. Is this easily fixable? 

![image](https://github.com/sn4k3/UVtools/assets/4185392/d3fb7a2a-62aa-4139-8468-8fbb1c9c0bd2)

Edit...my file is over the 25mb size limit, so I cannot attach it. If you need to see an example file, I'm sure I could make a dummy file that mimics this error. Let me know!

**Comment by sn4k3** [comment]
that print time presentation was never made to support more than 24h prints. I can change that to present days too on next patch.
